reframe/frontend/cli.py

```diff
@@ -903,13 +903,15 @@ def module_unuse(*paths):
         exec_policy.sched_flex_alloc_nodes = sched_flex_alloc_nodes
         parsed_job_options = []
         for opt in options.job_options:
-            optstr, *_ = re.split(r'=|\s+', opt)
+            opt_split = re.split(r'=|\s+', opt, maxsplit=1)
```

Comment:
Since we are sticking with `opt=val` syntax and we do the conversion, does it make sense to split also on spaces? In that cases, the user can always pass the full option `--opt val`. What do you think?